sdk/go/common/util/executable/executable.go

```diff
@@ -55,3 +64,15 @@ func FindExecutable(program string) (string, error) {
 
 	return "", errors.Errorf(unableToFindProgramTemplate, program)
 }
+
+func getPotentialPaths(goPath string, os string) []string {
```

Comment:
I believe this function could return a (err, string[]), because what happens if the os is not linux/darwin or windows? But at the same time, I checked in the README, and windows and linux/darwin are the only supported platforms.